packages/sabnzbd/default_paths.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/sabnzbd/constants.py b/sabnzbd/constants.py
+index 8f691015a..dbfa56b6a 100644
+--- a/sabnzbd/constants.py
++++ b/sabnzbd/constants.py
+@@ -60,8 +60,8 @@ CONFIG_BACKUP_FILES = [
+     DB_HISTORY_NAME,
+ ]
+ 
+-DEF_DOWNLOAD_DIR = os.path.normpath("Downloads/incomplete")
+-DEF_COMPLETE_DIR = os.path.normpath("Downloads/complete")
++DEF_DOWNLOAD_DIR = os.path.normpath("storage/downloads/incomplete")
```

Comment:
This path should not be `storage`, if I did not execute `termux-setup-storage`, then it will create a `storage` folder in the current directory, if I execute `termux-setup-storage` will delete it.
(Under the premise of running under the `$HOME`)

Reply:
Yeah, I was unsure about this one. It's still just default so users can edit it when setting up. I was thinking even about mentioning running `termux-setup-storage` at the end of installation. I removed this patch as it's indeed too obscure.